CSS/mob-style.scss

```diff
@@ -0,0 +1,295 @@
+@use 'variables' as *;
+
+.navbar {
+  padding: 1rem 1.5rem;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+
```

Comment:
- [ ] The mobile menu is not working on the about page in mobile view, Kindly fix this issue.

![image](https://user-images.githubusercontent.com/81932478/190178750-9f4c836e-253b-4f1d-b6ed-e91380855387.png)

Reply:
Thank you